src/components/Books.jsx

```diff
@@ -0,0 +1,25 @@
+/* eslint-disable react/prop-types */
+import Form from './Form';
+
+const Books = ({ title, author }) => (
+  <>
+    <div>
+      <ul>
+        <li>
+          Title :
+          {title}
+        </li>
+        <li>
+          Author :
+          {author}
+        </li>
+      </ul>
+      <button type="submit" className="remove">remove</button>
+    </div>
+    <div>
+      <Form />
+    </div>
+  </>
+);
```

Comment:
as required your form should have a submit placeholder button

Reply:
@alexander16108 
![requirments](https://user-images.githubusercontent.com/119133987/236577694-e754dab7-2e10-47d7-989c-36aabd5039d7.PNG)
Can you give me more details?

Reply:
Where exactly I'm supposed to add the **placeholder**?
@alexander16108